src/components/victory-area/victory-area.js

```diff
@@ -23,6 +23,42 @@ export default class VictoryArea extends React.Component {
   static role = "area";
 
   static defaultTransitions = {
+    onLoad: {
+      duration: 2000,
+      entrance: "left",
+      beforeClipPathWidth: (data, child, animate) => {
+        const paddingLeft = child.type.getScale(child.props).x.range()[0];
```

Comment:
If all you need is the range you can do `Helpers.getRange(child.props)` from victory-core